.github/workflows/sysdig-teams-operator-build-prod.yaml

```diff
@@ -0,0 +1,26 @@
+name: sysdig-teams-operator-build-pathfinder
+on:
+  push:
+    branches: 
+    - 'main'
+    paths: 
+    - 'operator/sysdig-monitor/**'
+    - '.github/workflows/sysdig-teams-operator-build-prod.yaml'
+ 
+jobs:
+  build: 
+    runs-on: ubuntu-latest
+    steps:
+    - name: Get the current branch name
+      shell: bash
+      run: echo "::set-output name=branch::${GITHUB_REF#refs/heads/}"
+      id: branchref
+    - uses: actions/checkout@v1
+      with:
+        ref: ${{ github.ref }}
+    - name: Login to DockerHub Registry
+      run: echo ${{ secrets.DOCKERHUB_PASSWORD }} | docker login -u ${{ secrets.DOCKERHUB_USERNAME }} --password-stdin
+    - name: Build the tagged Docker image
+      run: cd operator/sysdig-monitor && docker build . --file Dockerfile --tag bcdevopscluster/sysdig-teams-operator:prod
+    - name: Push the tagged Docker image
+      run: docker push bcdevopscluster/sysdig-teams-operator:prod
```

Comment:
At some point, these builds should be going into Artifactory instead of DockerHub.

Reply:
yes totally! i got a ticket to move from docker to artifactory. Gotta figure out how to do that on github actions first and the rest should be pretty straight forward!

Reply:
Can I get a demo/walkthrough once you figure it out?

Reply:
sure. if pushing to artifactory directly doesn't work then plan B is probably just to trigger a build in namespace.